gallery/src/components/NxThreatCounter/NxThreatCounterPage.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxThreatCounterExample from './NxThreatCounterExample';
+import NxThreatCounterColumnExample from './NxThreatCounterColumnExample';
+import NxThreatCounterGridExample from './NxThreatCounterGridExample';
+
+const NxThreatCounterCode = require('./NxThreatCounterExample?raw'),
+    NxThreatCounterColumnCode = require('./NxThreatCounterColumnExample?raw'),
+    NxThreatCounterGridCode = require('./NxThreatCounterGridExample?raw');
+
+const NxThreatCounterPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        Basic style for small counter which displays a number representing a count of violations, and a short pre-set
+        text string which displays the threat severity.
+      </p>
+      <p className="nx-p">
+        Three basic layouts have been demonstrated below. It is expected that one of these should satisfy any
+        requirement.
+      </p>
+      <table className="nx-table nx-table--gallery-props">
+        <thead>
+          <tr className="nx-table-row">
+            <th className="nx-cell nx-cell--header">Prop</th>
+            <th className="nx-cell nx-cell--header">Type</th>
+            <th className="nx-cell nx-cell--header">Required</th>
+            <th className="nx-cell nx-cell--header">Details</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr className="nx-table-row">
+            <td className="nx-cell">criticalCount | severeCount | moderateCount | lowCount</td>
+            <td className="nx-cell">numerical</td>
```

Comment:
Where possible we should stick to the actual type names in TS/JS
```suggestion
            <td className="nx-cell">number</td>
```